### PR TITLE
{numlib}[GCCcore/6.4.0] TetGen v1.5.0

### PR DESCRIPTION
**Pull request by smoors**

### CODE REVIEW

File: easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+easyblock = 'CMakeMake'
+
+name = 'TetGen'
+version = '1.5.0'
+
+homepage = 'http://wias-berlin.de/software/index.jsp?id=TetGen'
+description = """ A Quality Tetrahedral Mesh Generator and a 3D Delaunay Triangulator """
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
```
Comment (boegel) [inline easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb]: Any particular reason you're not building this with Intel compilers?

File: easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    'mkdir -p %(installdir)s/bin/ && cp tetgen %(installdir)s/bin/',
+    'mkdir -p %(installdir)s/lib/ && cp libtet.a %(installdir)s/lib/',
+]
+
+sanity_check_paths = {
+    'files': ['lib/libtet.a', 'bin/tetgen'],
+    'dirs': [],
+}
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb]: drop duplicate empty line please

File: easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+skipsteps = ['install']
+
+postinstallcmds = [
+    'mkdir -p %(installdir)s/bin/ && cp tetgen %(installdir)s/bin/',
+    'mkdir -p %(installdir)s/lib/ && cp libtet.a %(installdir)s/lib/',
+]
+
+sanity_check_paths = {
+    'files': ['lib/libtet.a', 'bin/tetgen'],
```
Comment (boegel) [inline easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb]: please sort alphabetically

File: easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    ('binutils', '2.28'),
+]
+
+separate_build_dir = True
+
+skipsteps = ['install']
+
+postinstallcmds = [
+    'mkdir -p %(installdir)s/bin/ && cp tetgen %(installdir)s/bin/',
+    'mkdir -p %(installdir)s/lib/ && cp libtet.a %(installdir)s/lib/',
```
Comment (boegel) [inline easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb]: Hmm, `make install` doesn't work as expected? That's a bit strange with `CMake` being used, no?

File: easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb]: It allows to use it with both the foss and intel chains (I intend to use it with foss).

File: easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb (hunk 4 above)
Comment (smoors) [reply easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb]: "make install" is not defined. You know a better way to handle this?

Comment (boegel) [reply easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb]: Isn't `cmake` supposed to make sure it *is* defined in the generated `Makefile`? @wpoely86?

Comment (wpoely86) [reply easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb]: yes and no. You still need to tell CMake what it is supposed to install and where. This `CMakeFile.txt` is very minimal and only has the instructions to build it. It's this or we patch the `CMakeFile.txt`.

Comment (boegel) [reply easybuild/easyconfigs/t/TetGen/TetGen-1.5.0-GCCcore-6.4.0.eb]: Ugh, ok fine, anything is better than patching the `CMakeFile.txt`. ;-)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f74b88b77f01659219e4c2107a344139 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2431.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2ec0bdcb3390ca80547736a1493a5d49 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**